Update Telemetry Docs (#23535)
hashicorp/vault: website/content/partials/telemetry-metrics/vault/replication/fsm/last_remote_wal.mdx

Metric type | Value   | Description
----------- | ------- | -----------

> gauge       | number  | Index of the last remote write-ahead log.  Note this is not emitted on standby nodes.

Reviewer: ```suggestion
gauge       | number  | Index of the last remote write-ahead log.

<Note>
  Standby nodes do not emit `last_remote_wal` details.
</Note>
```

Style correction: avoid "this" as a pronoun.
